### PR TITLE
chore: remove webpack aliases

### PR DESCRIPTION
**Pull request by Anemy**
This PR removes [aliases](https://webpack.js.org/configuration/resolve/#resolvealias) for modules in our webpack config. Did a little bit of formatting on imports as well, having node module imports at the top and local below separated by a line.

Do ya'll feel aliases should remain? I was thinking they don't really bring much to the table and both make navigation more difficult, and make it easier to misinterpret an import as a node_module rather than local file.

### CODE REVIEW

File: src/components/form/is-favorite-pill.spec.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+
```
Comment (alenakhineika) [inline src/components/form/is-favorite-pill.spec.js:3]: Also, I would like to discuss with you usage of empty lines thing :) I don't mind to adopt any code styling rules, just would like to have those rules and write similarly.

**Review by alenakhineika — COMMENTED** [review]
I don't have a strong opinion about aliases. Why do you think they make navigation more difficult?

As for me, you just write less dots :)

```import Actions from 'actions';```
```import Actions from '../../actions';```

File: src/components/form/connection-form.jsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
-import React from 'react';
+import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import Actions from 'actions';
+import React from 'react';
+
+import Actions from '../../actions';
```
[the remark this answers is not in the document]
Comment (Anemy) [reply src/components/form/connection-form.jsx:5]: The main thing I think is nice is differentiating node module imports from local imports. Here I separate them with a line. From there just a preference of mine is I like to make them alphabetical, I don't think it's any sort of requirement though.

**Comment by Anemy** [comment]
@alenakhineika when webpack is aliasing, it makes it harder for the editor to resolve where the dependency is going:
Screen Shot 2021-01-07 at 12 07 20 PM
Also, when developing, when using a module not yet defined in that file, vscode can see that import is in the workspace and automatically add it to the imports in that file. The added import then does not use the alias. We could probably make it so that it does use aliases, but I'm not sure I see anything that aliases add in this case.

**Comment by alenakhineika** [comment]
@Anemy ah I see now, well then we can get rid of these aliases 👍

**Pull request merged**